tailwind.config.ts

```diff
@@ -3,6 +3,9 @@ module.exports = {
   content: ['./src/**/*.{html,js,ts,jsx,tsx}'],
   theme: {
     extend: {
+      backdropBlur: {
```

Comment:
changed the backdrop blur value from 4px to 1.5px

Reply:
Okay